client/src/App.js

```diff
@@ -1,12 +1,13 @@
 import { Route, Routes } from "react-router-dom";
-import { Login, Signup } from "./Pages";
-import Home from "./Pages/Home";
+import { Login, Signup, Home } from "./Pages";
+import Landing from "./Pages/Landing";
```

Comment:
I think can just add Landing into the import statement above

Reply:
I TRIED INITIALLY but it doesnt work idk whyy :(